prov/gni/src/gnix_rma.c

```diff
@@ -1346,6 +1346,11 @@ ssize_t _gnix_rma(struct gnix_fid_ep *ep, enum gnix_fab_req_type fr_type,
 		return -FI_EINVAL;
 	}
 
+	/* Disallow fi_more with fi_fence */
```

Comment:
I think I'd word this differently.  It's really the case that a FI_FENCE breaks the FI_MORE chain.